### PR TITLE
AiSD-2-isPrime

### PR DESCRIPTION
**Pull request by mztezi**

### CODE REVIEW

File: isPrime.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+function isPrime(n) {
+    if (n === 2 || n === 3) {
+        return true;
+    }
+
+    if (n <= 1 || n % 2 === 0) {
+        return false;
+    }
+
+    let SqRoot = Math.sqrt(n);
```
Comment (kdryzek) [inline isPrime.js]: const sqrt = ...

File: isPrime.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+function isPrime(n) {
+    if (n === 2 || n === 3) {
```
Comment (kdryzek) [inline isPrime.js]: 3 is handled below

Comment (mztezi) [reply isPrime.js]: fixed

File: isPrime.js (hunk 1 above)
Comment (mztezi) [reply isPrime.js]: fixed

Comment (kdryzek) [reply isPrime.js]: const name still uppercased

**Review by kdryzek — APPROVED** [review]

**Pull request merged**